### ISSUE TITLE
Use the same `display_options` for columns present in tables, for explorations

### ISSUE THREAD

**Issue by kgodey** [centerofci/mathesar#1803]
Also ensure the `display_options` persist when aggregating values

**Comment by kgodey** [comment]
Blocked on #1805

**Comment by dmos62** [comment]
This seems to already be done on backend. Removing `work: backend` label. Readd if I misunderstood something.

**Comment by dmos62** [comment]
@pavish noted that #2008 is only part of the solution. I did not realise that we want display options to be inherited by new aliases. I can see how we can do this with some aliases, but not those generated via aggregation (without a lot of fuss).

I don't see how it makes sense for an aggregated column to inherit display_options from the column it was aggregated from. What the output of an aggregation will be depends on the aggregation function; it might be a different type (e.g. anything really, but often a number or an array).

Is the goal that frontend could style array items based on the display options for the column that was passed to an aggregation function to generate that array? What about other aggregation functions (i.e. functions other than `aggregate_to_array`)?

So far I've considered any kind of "inheritance" to be broken when you aggregate. Allowing inheritance through aggregation would be quite fiddly. I suggest postponing this feature. It requires an order of magnitude more work than inheritance for grouped columns.

**Comment by pavish** [comment]
@dmos62 I don't know if we can postpone this because formatting is important for types like dates and number, which will be displayed as arrays when aggregated. This would probably be needed for the demo video.

We do not have to worry about other functions, and the focus is currently only on the `aggregate_to_array` function.

@kgodey We need your thoughts on this.

**Comment by dmos62** [comment]
@pavish gotcha.

I'm in favor of sidestepping this somehow. Installing special treatment for `aggregate_to_array` would be not easy and technically indebting: preferably we'd focus on some more general mechanism at some point down the road.

**Comment by kgodey** [comment]
We should not postpone this. I'm sure we can find a quick solution.

**Comment by dmos62** [comment]
Found a way to do this quickly, without much of a fuss, in #2056. Sorry for misleading expectations.
